drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -860,7 +860,7 @@ uint32_t RenderingDeviceVulkan::get_image_format_pixel_size(DataFormat p_format)
 		case DATA_FORMAT_D24_UNORM_S8_UINT:
 			return 4;
 		case DATA_FORMAT_D32_SFLOAT_S8_UINT:
-			return 5; //?
+			return 5; // ?.
```

Comment:
This comment can probably just be removed...?

Reply:
I believe the comment has value (it's not clear whether that's the right value to return in that case). However, a period is redundant after a question mark, so I've kept the question mark alone.